clients/dashboards/dashboards.go

```diff
@@ -0,0 +1,82 @@
+package dashboards
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+	"github.com/influxdata/influx-cli/v2/pkg/influxid"
+)
+
+type Client struct {
+	clients.CLI
+	api.DashboardsApi
+	api.OrganizationsApi
+}
+
+type Params struct {
+	clients.OrgParams
+	Ids []string
+}
+
+func (c Client) List(ctx context.Context, params *Params) error {
+	orgID, err := c.getOrgID(ctx, params.OrgID, params.OrgName)
```

Comment:
It looks like `GetDashboards` supports querying by org name directly. Instead of this org ID lookup, I'd make the `req`-building code below use either `OrgID` or `Org` depending on the passed parameters. For example: https://github.com/influxdata/influx-cli/blob/abe521add0c20d91499e353e57c087d1bd56c431/clients/bucket/list.go#L23-L31

The difference can be significant based on the permissions assigned to the user running the command.